docs/content/crds/generate.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/rotisserie/eris"
+	"github.com/solo-io/gloo/pkg/utils/cmdutils"
+	"github.com/solo-io/gloo/projects/gateway2/api/v1alpha1"
+	"github.com/solo-io/skv2/codegen/util"
+	"github.com/stoewer/go-strcase"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	// crdocBinary is assumed to be installed locally to this path
+	// see the `install-go-tools` target in the Makefile
+	crdocBinary = filepath.Join(util.GetModuleRoot(), "_output", ".bin", "crdoc")
```

Comment:
I'm not sure how we woudl do that. Could you clarify?

Reply:
I wanted to make this as simple as possible and avoid adding arguments. I think in the future we could make it work that way, but if feels easier to make an assumption aabout a binary location, rather than require callers to provide it

Reply:
Yeah, I mean pass as an argument or an env var. But fine with me. And yeah, easy enough to amend in the future if we choose.